src/test/staticFiles/workspace/.gitignore

```diff
@@ -0,0 +1,4 @@
+.codechecker/
+.codechecker-alt/
+.vscode/settings.json
+file
```

Comment:
`runTest.ts` has a `CodeChecker log` run, which runs `c++`, which creates this file.

I don't know of a way to run gcc without output, with CodeChecker log being able to create a compdb, but with that, this could be removed.

Reply:
`-o /dev/null` or perhaps GCC also support `-fsyntax-only`? The latter one might be a Clang specific flag.

Reply:
`-o /dev/null` worked well enough, thanks!